src/language-js/utils/is-node-matches.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
Though this function was written by me (fisker), and I'm fine to use it in this repo, but maybe we should add source here? 


```suggestion
"use strict";

// Copied from https://github.com/sindresorhus/eslint-plugin-unicorn/blob/d53d935951aa815c763fc9441aa452c763294715/rules/utils/is-node-matches.js
```

Reply:
Exactly right. I'm sorry.
It lacked consideration.